contracts/0.4.24/interfaces/ILido.sol

```diff
@@ -131,4 +131,7 @@ interface ILido {
       * @return beaconBalance - total amount of Beacon-side Ether (sum of all the balances of Lido validators)
       */
     function getBeaconStat() external view returns (uint256 depositedValidators, uint256 beaconValidators, uint256 beaconBalance);
+
+    event SetAllowedBeaconBalanceAnnualRelativeIncrease(uint256 value);
+    event SetAllowedBeaconBalanceAnnualRelativeDecrease(uint256 value);
```

Comment:
Wrong interface. We need ILidoOracle.sol :)

Reply:
done